orange-pi-zero.coffee

```diff
@@ -0,0 +1,41 @@
+
+deviceTypesCommon = require '@resin.io/device-types/common'
+{ networkOptions, commonImg, instructions } = deviceTypesCommon
+
+postProvisioningInstructions = [
+	instructions.BOARD_SHUTDOWN
+	instructions.REMOVE_INSTALL_MEDIA
+	instructions.BOARD_REPOWER
+]
```

Comment:
I checked the specs for the Orange Pi Zero and apparently it lacks eMMC. So it will only run from SD card. In which case, the post provisioning instructions are not needed. They need to be removed. Can you amend this commit and remove them from this commit please?

Reply:
Sure thing, sorry for that.

Reply:
Done